google/cloud/documentai_toolbox/wrappers/hocr.py

```diff
@@ -0,0 +1,221 @@
+import dataclasses
+from typing import List, Union
+from google.cloud import documentai
+
+from google.cloud.documentai_toolbox.constants import ElementWithLayout
+
+
+def _get_bounding_box(element_with_layout: ElementWithLayout, dimensions: documentai.Document.Page.Dimension):
+    if element_with_layout.layout.bounding_poly.vertices:
+        min_x, min_y = element_with_layout.layout.bounding_poly.vertices[0].x, element_with_layout.layout.bounding_poly.vertices[0].y
+        max_x, max_y = element_with_layout.layout.bounding_poly.vertices[2].x, element_with_layout.layout.bounding_poly.vertices[2].y
+        return f"bbox {int(min_x)} {int(min_y)} {int(max_x)} {int(max_y)}"
+    min_x, min_y = element_with_layout.layout.bounding_poly.normalized_vertices[0].x, element_with_layout.layout.bounding_poly.normalized_vertices[0].y
+    max_x, max_y = element_with_layout.layout.bounding_poly.normalized_vertices[2].x, element_with_layout.layout.bounding_poly.normalized_vertices[2].y
+    return f"bbox {int(min_x * dimensions.width)} {int(min_y * dimensions.height)} {int(max_x * dimensions.width)} {int(max_y * dimensions.height)}"
+
+
+def _get_text(element_with_layout: ElementWithLayout, document_text):
+    start_index = element_with_layout.layout.text_anchor.text_segments[0].start_index
```

Comment:
Similarly here, we should consider either 

(1) promoting `ElementWithLayout` to a proper class (and not just a type alias) so that we can put the code `element_with_layout.layout.text_anchor.text_segments[0].start_index` into an `ElementWithLayout.start_index` property.  (And in fact, then, `_get_text` could be just a property of that class.)

or 

(2) introduce a private wrapper class of `Layout` or `TextAnchor`.

(This refactoring appears to be purely internal and does not have to block the hOCR export work.)

Reply:
Yea that's something we can do to simplify our usage of these across the toolbox, I think it might be beneficial to add this change in a separate PR and not the PR for hOCR feature.